src/css/controls/imports/display.less

```diff
@@ -83,6 +83,12 @@ display icons
         display: none;
     }
 
+    &.jw-flag-touch .jw-display .jw-icon,
+    &.jw-flag-touch .jw-display .jw-svg-icon {
+        z-index: 100;
+        position: relative;
+    }
+
```

Comment:
I brought the play/pause button to the front so it takes precedent

Reply:
no i think it makes sense in the PR! would clutter the file for a specific fix otherwise